book/standard_library.md

```diff
@@ -50,83 +52,49 @@ See the [notes below](#optimal-startup) on how to ensure that your configuration
 
-2. In the `use std/help` example, `help` is both the submodule name and the name of the main command it exports. Using `use std/help` allows you to access:
-
-   ```nu
-   help
-   help aliases
-   help modules
-   help commands
-   help operators
-   help externs
-   ```
-
-   In this case, the `std/help` commands are meant to shadow (replace) the built-in versions.
-
-Submodules that are normally imported with `use std/<submodule>`:
+These submodules are normally imported with `use std/<submodule>` (without a glob/`*`):
 
 - `use std/assert`: `assert` and its subcommands
 - `use std/bench`: The benchmarking command `bench`
 - `use std/dirs`: The directory stack command `dirs` and its subcommands
 - `use std/input`: The `input display` command
 - `use std/help`: An alternative version of the `help` command and its subcommands which supports completion and other features
-- `use std/iters`: Additional `iters`-prefixed iteration commands. Note: See-also alternative option #3 below.
+- `use std/iters`: Additional `iters`-prefixed iteration commands.
 - `use std/log`: The `log <subcommands>` such as `log warning <msg>`
-- `use std/math`: Mathematical constants such as `$math.E`. These can also be imported without a prefix using Form #2 below.
+- `use std/math`: Mathematical constants such as `$math.E`. These can also be imported as definitions as in Form #2 below.
 
-#### 2. Import the _contents_ of the module directly
+#### 2. Import the _definitions_ (contents) of the module directly
 
-For certain submodules, you will want the commands from a submodule to be available in the current scope, so that you _can_ simply access the command by name. For instance:
+Some submodules are easier to use when their definitions (commands, aliases, constants, etc.) are loaded into the current scope. For instance:
 
 ```nu
 use std/formats *
 ls | to jsonl
 ```
 
-Submodules that are normally imported with `use std/<submodule> *`:
+Submodules that are normally imported with `use std/<submodule> *` (**with** a glob/`*`):
 
 - `use std/dt *`: Additional commands for working with `date` values
 - `use std/formats *`: Additional `to` and `from` format conversions
 - `use std/math *`: The math constants without a prefix, such as `$E`. Note that the prefixed form #1 above is likely more understandable when reading and maintaining code.
 - `use std/xml *`: Additional commands for working with XML data
 
-#### 3. Importing specific subcommands
+#### 3. `use std <submodule>`
 
-As with most modules, you can choose to import only a subset of the commands. For instance, the following would import the `zip-with` command without requiring that it be called with`iter zip-with`.
-
-```nu
-use std/iter [ zip-with ]
-```
-
-#### 4. `use std <submodule>`
-
-While it is _possible_ to import Standard Library submodules using a space-separated form:
+It is _possible_ to import Standard Library submodules using a space-separated form:
```

Comment:
This form isn't mentioned in "Using Modules", so this line implies that this form is only possible for the stdlib, which isn't really true. I think it is probably worth mentioning in "Using Modules", even if we explicitly recommend against it.

Reply:
It's covered in the bullet *"Import one or more commands from a module"*, which I've renamed *"Import one or more definitions from a module"*.  Below that, it calls out that a definition can be *"Other modules (submodules)"*.  I've expanded it with a note on that form specifically and also added a pointer here in the Standard Library chapter back to it.